keyboards/crkbd/keymaps/md40/keymap.c

```diff
@@ -0,0 +1,33 @@
+#include QMK_KEYBOARD_H
```

Comment:
This file is missing a license header.

Reply:
Could you add a GPL2+ compatible license header here?  All of the source files need to have a license explicitly listed.

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Reply:
Added, thanks!